### PR TITLE
DEVX-1648: Add C3's role binding view to cp-demo

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-1648

_What behavior does this PR change, and why?_

Add RBAC role bindings screenshot

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation (validated with local build)

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

- [ ] Documentation

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -633,6 +641,10 @@ End clients (non-CP clients):
 
          kafka1            | 	super.users = User:admin;User:mds;User:superUser;User:ANONYMOUS
 
+#. From the |c3| UI, in the Administration menu, click the *Manage role assignments* option. Click on ``Assignments`` and then the Kafka cluster ID. From the ``Topic`` list, verify that the LDAP user ``appSA`` is allowed to access a few topics, including any topic whose name starts with ``wikipedia``. This role assignment was done during ``cp-demo`` bringup in the :devx-cp-demo:`create-role-bindings.sh script|scripts/helper/create-role-bindings.sh`.
```
Comment (mikebin) [inline docs/index.rst:644]: > during cp-demo bringup ...

Maybe "startup" instead of "bringup" would read more clearly?

**Review by mikebin — APPROVED** [review]
LGTM - one trivial comment

File: docs/index.rst (hunk 1 above)
Comment (rspurgeon) [reply docs/index.rst]: @ybyzek I modified this bit based on @mikebin 's feedback and pushed a change to this branch.  I also slightly modified the associated doc image to remove some extraneous pixels around the border.

**Review by rspurgeon — APPROVED** [review]
LGTM

File: docs/index.rst (hunk 1 above)
Comment (ybyzek) [reply docs/index.rst]: Thanks @rspurgeon !

**Pull request merged**